app/src/sharedTest/java/org/oppia/android/app/walkthrough/WalkthroughTopicListFragmentTest.kt

```diff
@@ -12,7 +12,9 @@ import androidx.test.espresso.action.ViewActions.scrollTo
 import androidx.test.espresso.assertion.ViewAssertions.matches
 import androidx.test.espresso.contrib.RecyclerViewActions.scrollToPosition
 import androidx.test.espresso.intent.Intents
+import androidx.test.espresso.matcher.ViewMatchers
```

Comment:
we should not import `ViewMatchers ` as a whole but we can import whatever things we are using. 

Example `import androidx.test.espresso.matcher.ViewMatchers.isRoot`

Reply:
Done

Reply:
@anandwana001 To resolve this, I would have to also make changes in the other test cases of the file.